include/aca_vlan_manager.h

```diff
@@ -60,20 +67,22 @@ class ACA_Vlan_Manager {
                               alcor::schema::NetworkType network_type, string remote_host_ip,
                               uint tunnel_id, ulong &culminative_time);
 
-  // create a neighbor port without specifying vpc_id and neighbor ID
-  int create_neighbor_outport(alcor::schema::NetworkType network_type, string remote_host_ip,
-                              uint tunnel_id, ulong &culminative_time);
-
   int delete_neighbor_outport(string neighbor_id, uint tunnel_id,
                               string outport_name, ulong &culminative_time);
 
   int get_outports_unsafe(uint tunnel_id, string &outports);
 
-  void set_aux_gateway(uint tunnel_id, const string auxGateway_id);
+  void set_auxgateway(uint tunnel_id, const string auxGateway_id, uint oam_port);
```

Comment:
we are making auxgateway=zeta gateway in the current implementation. auxgateway can include zeta and internet and NAT gateway at the same time. Let's focus the current change for zeta gateway and move all the functions to zeta files.